main.yml

```diff
@@ -4,6 +4,23 @@
 # Note, 'local' (the host name in the inventory)
 #   should be assigned ``ansible_connection=local``.
 
+# Ansible requires python2, while Ubuntu >=15 strictly uses Python3.
```

Comment:
I also did a fresh install of ubuntu server 16.04.1 (http://releases.ubuntu.com/16.04.1/ubuntu-16.04.1-server-amd64.iso) and it did not have `python` installed.